### DISCUSSION TITLE
enh: OAUTH_LOGO variable support

### DISCUSSION THREAD

**Discussion by Lanhild** [open-webui/open-webui#19352] [Ideas]
**Is your feature request related to a problem? Please describe.**
It would be nice to specify a custom logo for the OAuth provider.

**Describe the solution you'd like**
Dispose of a `OAUTH_LOGO` variable or the likes to input an image URL in order to have a custom logo displayed for the OAUTH button 
![image](https://github.com/user-attachments/assets/94664fff-4e0d-4f04-955f-acc2b3f97922)

**Comment by silentoplayz** [comment]
Does the environment variable `OAUTH_PICTURE_CLAIM` close this issue?
For reference - https://docs.openwebui.com/getting-started/advanced-topics/env-configuration#oauth_picture_claim

**Comment by Lanhild** [comment]
> Does the environment variable `OAUTH_PICTURE_CLAIM` close this issue?
> For reference - https://docs.openwebui.com/getting-started/advanced-topics/env-configuration#oauth_picture_claim

The variable you reference is used to select which claim is used from the payload for the user profile picture.
(A different thing than this issue)
